src/include/OSL/wide.h

```diff
@@ -11,6 +11,7 @@
 #include <OSL/dual_vec.h>
 #include <OSL/mask.h>
 #include <OSL/oslconfig.h>
+#include <OSL/oslclosure.h>
```

Comment:
Does this include need to be here?  (as there is no longer a reference to any of it's types in wide.h)

Reply:
Removed.